### PR TITLE
Readme updates and minor bugfixes

### PR DESCRIPTION
**Pull request by benallfree**

### CODE REVIEW

**Comment by otakustay** [comment]
Thanks for pull request

It is a little confused that we need to add `./node_modules/.bin/` in `package.json/scripts`, `npm run` should automatically add this to `PATH` env, can you provide a simple reproduce steps to show how npm scripts are not working?

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ For best display effect, you should generate your diff text with `git diff -U1`
 
 The `{File[] parseDiff({string} text, {Object} [options])` named export is a wrap of [parse-diff](https://www.npmjs.com/package/parse-diff) package with some extra options:
 
-- `{boolean} stubChunk`: Whether to add a stub empty chunk at the tail of each chunk list, this can provide an extra hunk header when [customizing chunk header](#customize-chunk-header), for example, to expand code after the diff.
+- `{boolean} stubChunk`: Whether to add a stub empty hunk at the tail of each hunk list, this can provide an extra hunk header when [customizing hunk header](#customize-hunk-header), for example, to expand code after the diff.
```
Comment (otakustay) [inline README.md]: There are still have some `chunk` -> `hunk` fixes left, can you just global replace `Chunk` to `Hunk` (first letter upper cased) in `README.md`

**Comment by otakustay** [comment]
Thanks, however it's not likely to publish a `1.0.0` version currently, I'm still developing a faster and more powerful `parseDiff` function with rename/copy detection, chmod detection and binary detection, if you'd like to use it immediately, I can publish a `0.x` beta version

**Comment by benallfree** [comment]
Hey ok readme fixed, and yes the `package.json` mods turned out to be a `yarn` bug, because `npm install` works fine, so I reverted them.

Another issue I noticed...not sure what to do about this... It seems like the `index.map` file is not used/honored when I `require` this package in my React frontend. So debugging gets difficult.

FYI I'm using your package in this tool: https://github.com/benallfree/multi-repo-differ which helps me track changes when a feature spans multiple repos.

**Comment by benallfree** [comment]
I think it's definitely ready to be used. I used it :)  I say publish a `0.x` version. Thank you for doing all this work, it's a really cool project.

**Comment by otakustay** [comment]
I published a `0.8.0` version to npm, will dive into the source map issue tomorrow :)

**Comment by otakustay** [comment]
@benallfree I explored the source map issue, it appears that since `react-diff-view` places source map as a separated file aside the `.js` bundle, we need to use [source-map-loader](https://github.com/webpack-contrib/source-map-loader) to locate source map and provide it to webpack

Currently I've got it to work with source-map-loader in Chrome 62.0.3202.29, if you have any further problem please fire an issue so I can get notified and track it

**Pull request merged**